app/components/questionnaire/answer-form-list.js

```diff
@@ -11,6 +11,15 @@ const ComponentInfos = [
   }
 ];
 
+const ComponentSettings = {
```

Comment:
Seeing this here, I think it would be better to combine `ComponentInfos` and `ComponentSettings` into one object, since they're linked anyways. The name `ComponentSettings` is better anyways. Some suggested changes:

```
const ComponentSettings = [
   {
     name: 'file-group-list',  // Component to render
     cwlType: { type: 'array', items: { type: 'array', items: 'File' } }, // From CWL
     formats: [
       { name: 'FASTQ', // For displaying in the file picker if necessary
          format: 'http://edamontology.org/format_1930', 
          pattern: '.*(fq$)|(fastq$)|(fastq.gz$)', 
          groupTitle: 'sample' }
     ]
    }
```

Might be cleaner to move this out to another file too.